account-kit/react/src/components/auth/card/add-passkey.tsx

```diff
@@ -49,7 +49,7 @@ export const AddPasskey = () => {
         {BENEFITS.map(({ title, icon: Icon, description }) => (
           <div key={title} className="flex gap-2">
             <div className="h-5 w-5 flex items-center justify-center">
-              <Icon illustrationStyle={illustrationStyle ?? "flat"} />
+              <Icon illustrationStyle={illustrationStyle} />
```

Comment:
@moldy530 if you'd prefer I can move reading this from the hook into the icon components; it felt a bit weird to have icon components that read from a context to configure themselves but it could also be... not that weird, defer to you

Reply:
yea I was thinking about that too, but since all of the components we're building are meant to be used internally I think it's ok since it removes a good bit of boilerplate